### ISSUE TITLE
Aria labeling functionallity not working

### ISSUE THREAD

**Issue by Kyuseishun** [jfangrad/react-dropdown-aria#43]
I'm developing an accessible page but tests with NVDA show that Aria tags are not working properly, might I ask what screen reader software was this tested on?

**Comment by jfangrad** [comment]
It was tested using the kinda crappy ChromeVox extension (Developing on Ubuntu so no NVDA :disappointed:). Can you be a bit more specific about what isn't working properly?

**Comment by Kyuseishun** [comment]
the virtual cursor is standing on the list and points out the number of options available, but it doesn't read the aria-label tag in the input element, also ariaLabeledBy and ariaDescribedby don't have any effect while using NVDA for the same reason

**Comment by AndrewEastwood** [comment]
looks like ariaLablel is being ignored and each option is using 'value' key....
